### PR TITLE
Adds patternfly and font awesome icons

### PR DESCRIPTION
**Pull request by andresgalante**
I am adding icons, please review this with care since I've wrote documentation and I suck at writing in english.
#11

### CODE REVIEW

File: source/_patterns/basics/03-icons/00-icon.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Buttons
```
Comment (cshinn) [inline source/_patterns/basics/03-icons/00-icon.md]: Was this file intended to be edited?

File: source/_patterns/basics/03-icons/00-icon.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+---
+## Overview
+
+If this is a bootstrap component:
+This is a Bootstrap component, [see complete documentation here](URL)
```
Comment (srambach) [inline source/_patterns/basics/03-icons/00-icon.md]: Did you mean to update this file with icon info?

**Comment by andresgalante** [comment]
@cshinn @srambach Please review again, I am sorry I missed the md files

**Comment by cshinn** [comment]
:shipit:

**Comment by cshinn** [comment]
LGTM! what do you think @srambach?

**Comment by srambach** [comment]
+1
